src/spec/dsl.cr

```diff
@@ -175,6 +175,7 @@ module Spec
   # ```
   #
   # will print, just before each spec, 1 and then 2.
+  @[Deprecated("Use `Spec::Methods.before_each` instead")]
```

Comment:
The message should suggest to use the top-level before_each. Users of spec are not neccesarily familiar with it's design and the fact there is a `include Spec::Methods` somewhere in the root scope.

Reply:
I know this is not ideal, but there is no other way to address the replacement method. In ``Use `::before_each` instead``, the method name wouldn't be linked. This is an issue of `Spec::Methods`, not of the deprecation message.